trino/client.py

```diff
@@ -127,6 +131,7 @@ def __init__(
         self._roles = roles.copy() if roles is not None else {}
         self._prepared_statements: Dict[str, str] = {}
         self._object_lock = threading.Lock()
+        self._timezone = timezone or get_localzone_name()
```

Comment:
This might be a breaking change, In the sense that suddenly the local timezone will be used compared to UTC.

I'll leave this to the maintainers to decide if we need to put it behind a feature flag or announce it as a breaking change.

Reply:
Ok, how should we proceed?

Reply:
Q: Before this change the server returned values in UTC? If yes, then won't setting `timezone=UTC` here explicitly preserve old behaviour? If yes then it means a workaround exists to preserve old behaviour and we can mention it in release notes.

Note that in larger picture this makes Python client match JDBC client etc. So I'd consider it a bugfix which happens to be breaking change.

Reply:
The timezone returned by Trino CLI is the local timezone.